web/sites/default/config/webform.webform.report_row_obstruction.yml

```diff
@@ -384,11 +371,11 @@ elements: |-
             - or
             - ':input[name="report_signage_type"]':
                 value: 'A-board sign'
-      contact_email:
+      contact_home_address:
         '#type': textfield
-        '#title': Email
-        '#pattern': '^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$'
-        '#pattern_error': 'Please enter a correctly formatted email address, such as myname@example.com.'
+        '#title': 'Home Address'
```

Comment:
It's not clear to me whether this is supposed to be *my* home address or e.g. the address of the home that the issue is located at. Should we add something in the description?

also, is this supposed to be a textfield or the address widget?

Reply:
I've updated the label to "Your Home Address" to clarify. Decided to just use a textfield instead of the widget with Josh's approval. I need to make improvements to the widget before we start using it more.